src/app/tests/suites/certification/Test_TC_DL_2_2.yaml

```diff
@@ -11,73 +11,166 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
 
-name: 100.2.2. [TC-DL-2.2] Verification for Door lock command[DUT-Server]
+name: 108.2.2. [TC-DL-2.2] Verification for Door lock command[DUT-Server]
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic"
-    endpoint: 0
+    cluster: "Door Lock"
+    endpoint: 1
 
 tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
     - label:
           "TH writes the RequirePINforRemoteOperation attribute value as False
           on the DUT"
-      verification: |
-          https://github.com/project-chip/connectedhomeip/issues/15776
-      disabled: true
-
-    - label: "TH sends Lock Door Command to the DUT without any valid PINCode"
-      verification: |
-
-      disabled: true
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: false
+
+    - label: "TH sends Lock Door Command to the DUT without PINCode"
+      command: "LockDoor"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: ""
+      response:
+          error: FAILURE
```

Comment:
Why, if RequirePINforRemoteOperation is false?

Reply:
pls refer spec screenshot here , which indicates that if a PIN is provide, it will have to be valid regardless of RequirePINForRemoteOperation.
![2022-05-12_09-07](https://user-images.githubusercontent.com/57476670/167987214-cf24d747-629a-4daa-aa14-f0bd7af6d569.png)

Testplan Screenshot
![2022-05-12_09-03](https://user-images.githubusercontent.com/57476670/167987290-d1c4119a-0fd6-41c0-bf4e-2d4aa52d7b53.png)